qiita_pet/handlers/api_proxy/tests/test_artifact.py

```diff
@@ -97,8 +100,9 @@ def test_artifact_get_req(self):
     def test_artifact_post_req(self):
         # Create new prep template to attach artifact to
         new_prep_id = get_count('qiita.prep_template') + 1
-        PrepTemplate.create(pd.DataFrame(
-            {'new_col': {'1.SKD6.640190': 1}}), Study(1), '16S')
+        npt.assert_warns(
```

Comment:
Not correct, we are not interested in the warning message thrown here. the only reason why this is wrapped in the npt.assert_warns is because we don't want the warning message to show up on the test suite.